src/Core/Checkout/Customer/SalesChannel/UpsertAddressRoute.php

```diff
@@ -145,6 +145,7 @@ public function upsert(?string $addressId, RequestDataBag $data, SalesChannelCon
             'phoneNumber' => $data->get('phoneNumber'),
             'additionalAddressLine1' => $data->get('additionalAddressLine1'),
             'additionalAddressLine2' => $data->get('additionalAddressLine2'),
+            'customFields' => $data->get('customFields')->all(),
```

Comment:
I am not fan about allowing the actual client to pass anything to the customFields without server validation

My drawbacks:

- We don't have any validation of the user content. The user can send us anything, and we save it. The only limitation is the `max_allowed_packet ` from MySQL, which is in default 32MB
- The user can write into Extension defined customFields like maybe erpId, this address without tax etc. Idk what people use customFields for their purpose. Until this change they are private and the user cannot modify them